projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta.py

```diff
@@ -39,6 +42,18 @@ def details_job(self, job_name: str, team_name: str, execution_id: str) -> dict:
         """
         pass
 
+    @abc.abstractmethod
```

Comment:
I would probably say that they are sorted such that the last element in the list is the latest execution.
That seems counter intuitive to me. I would expect for the 0th element to be the last. execution.

Reply:
Updated the doc. Tested the Control Service API with different jobs and it constantly returns the most recent execution as the last element of the list.
Didn't want to do reverse list operation as it would have been yet another operation which seemed unnecessary.